frontend/src/pages/Game/Quizz/index.tsx

```diff
@@ -1,91 +1,73 @@
-import React, { useEffect, useState } from "react";
-import dancers from "../../../utils/dancers";
-import axios from "axios";
-import "./quizz.css";
-import { CookieKey, isTokenValid } from "../../../utils/auth";
-import Cookies from "universal-cookie";
+import React, { useState, useEffect } from "react";
+import Question from "./Question";
+import { IGameEnum } from "../types";
 
-const cookies = new Cookies();
+interface IQuizz {
+  incrementScore: () => void;
+  setStep: React.Dispatch<React.SetStateAction<IGameEnum>>;
+}
 
-const Quizz: React.FC<{ choices: string[] }> = ({ choices }) => {
-  const [dancer] = useState<string>(
-    dancers[Math.floor(Math.random() * dancers.length)]
-  );
-  const [track, setTrack] = useState<string | null>(null);
-  const [isPlaying, setIsPlaying] = useState<boolean>(true);
+interface ITracks {
+  track: string;
+  choices: string[];
+}
+
+interface IRequest {
+  data: ITracks[];
+  complete: boolean;
+  error: boolean;
+}
+
+const Quizz: React.FC<IQuizz> = ({ incrementScore, setStep }: IQuizz) => {
+  const [tracks, setTracks] = useState<ITracks[] | null>(null);
+  const [indexTrack, setIndexTrack] = useState<number>(0);
 
   useEffect(() => {
-    (async (): Promise<void> => {
-      if (isTokenValid()) {
-        const res = await axios.get(
-          "https://api.spotify.com/v1/search?q=muse&type=track&market=FR",
-          {
-            headers: {
-              Authorization: `Bearer ${cookies.get(CookieKey.ACCESS_TOKEN)}`
-            }
-          }
-        );
-        console.log(res.data.tracks.items[2]);
-        setTrack(res.data.tracks.items[2].preview_url);
-      }
-    })();
-  }, []);
+    // TODO: Request to get tracks and choices
+    const request: IRequest = {
+      data: [
+        {
+          track: "Hello world !",
+          choices: ["1", "2", "3", "4"]
+        },
+        {
+          track: "Hello world ! v2",
+          choices: ["1.2", "2.2", "3.2", "4.2"]
+        }
+      ],
+      complete: true,
+      error: false
+    };
 
-  const onPlayClicked = (): void => {
-    (document.getElementById("player") as HTMLAudioElement).play();
```

Comment:
Same here, maybe we should try to use the UseRef hook.

Reply:
This was deleted